### PR TITLE
Setup and documentation on running this on my local machine + bonus chroma on AWS Fargate (optional)

### PR DESCRIPTION
**Pull request by janhenrik**
I made this all work locally!

My version does not have the populate-script, it is embedded in get_cv.py.

See if you like it!

### CODE REVIEW

**Comment by A-Stangeland** [comment]
1. Documentation looks great, very much appreciated! 
2. Cool that got it to deploy on AWS, I will review the code when I get time. I am not yet decided how we're going to structure the project, so we may wait a bit before adding this.
3. What is the reason for removing Poetry? I think poetry makes it easier to have reproducible environments compared to just using pip.

Thank you for the contribution!

**Comment by janhenrik** [comment]
My bad. We should keep poetry, I will try to make it work with poetry before we merge - if you do not have the time.

File: infrastructure/chroma-db-fargate/bin/python
Comment (A-Stangeland) [inline infrastructure/chroma-db-fargate/bin/python:1]: Same as above, local path

File: infrastructure/chroma-db-fargate/bin/pip [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/Users/jhg/src/cvhelper/infrastructure/chroma-db-fargate/bin/python
```
Comment (A-Stangeland) [inline infrastructure/chroma-db-fargate/bin/pip:1]: This looks like a local path. Should probably not be included

**Review by A-Stangeland — CHANGES_REQUESTED** [review]
Docuentation looks good. I don't know much about AWS infrastructure, but I saw some local paths that should be removed. I think we can merge in the updates to the documentation now, and create a new PR for the infrastructure code when it is ready.

File: infrastructure/chroma-db-fargate/bin/pip (hunk 1 above)
Comment (janhenrik) [reply infrastructure/chroma-db-fargate/bin/pip:1]: Agree!

**Comment by janhenrik** [comment]
Just remove the Infrastructure-folder/aws stuff, I will continue on that later.

**Pull request merged**